### DISCUSSION TITLE
Is it possible to use some packages of adonis separately from the whole framework?

### DISCUSSION THREAD

**Discussion by cawa-93** [adonisjs/core#3724] [Help] [answered]
Let's say I only want to use adonisjs/lucid For working with databases and adonisjs/auth for authorization. But with another framework. 

Is it possible? What are the basic steps to get started?

**Comment by RomainLanz** [comment, marked as answer]
Hey! @cawa-93 

No, it is not doable.
If you want the power of AdonisJS, then use AdonisJS.
